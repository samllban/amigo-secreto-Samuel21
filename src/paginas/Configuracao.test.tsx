import { render } from "@testing-library/react";
import React from "react";
import { RecoilRoot } from "recoil";
import Configuracao from "./Configuracao";

const mockNavegate = jest.fn();

jest.mock('react-router-dom', () => {
    return {
        useNavigate : () => mockNavegate
    }
});

describe('A pagina de configuração', () => {
    test('Deve ser rendenrizada corretamente', () => {
        const { container } = render(<RecoilRoot>
            <Configuracao />
        </RecoilRoot>)
        
        expect(container).toMatchSnapshot();

    });
});