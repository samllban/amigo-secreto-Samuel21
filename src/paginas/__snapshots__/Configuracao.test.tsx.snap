// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`A pagina de configuração Deve ser rendenrizada corretamente 1`] = `
<div>
  <div
    class="card"
  >
    <section>
      <h2>
        Vamos começar!
      </h2>
      <form>
        <div
          class="grupo-input-btn"
        >
          <input
            placeholder="Insira os nomes dos participantes"
            type="text"
            value=""
          />
          <button
            disabled=""
          >
            Adicionar
          </button>
        </div>
        
      </form>
      <ul />
      <footer>
        <button
          disabled=""
        >
          Iniciar Brincadeira
        </button>
      </footer>
    </section>
  </div>
</div>
`;
